.pipelines/containerSourceData/scripts/BuildGoldenDistrolessContainer.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+set -e
+
+function DockerBuild {
+    local containerName=$1
+    local azureLinuxVersion=$2
+    local imageType=$3
+    local packagesToInstall=$4
+    local packagesToHoldback=$5
+    local installNonrootUser=$6
+    local rpmsDir=$7
+    local user=root
+    local userUid=0
+
+    if $installNonrootUser; then
+        user="nonroot"
+        userUid=65532
+    fi
+
+    # Create container
+    echo "+++ Create container $containerName"
+    docker build . \
+        -t "$containerName" \
+        -f "$marinaraSrcDir/dockerfiles/dockerfile-new-image" \
+        --build-arg MARINER_VERSION="$azureLinuxVersion" \
```

Comment:
Should this be AZL_VERSION instead of MARINER_VERSION?

Reply:
Yes, however this is the build argument in the marinara repo, which will need an update first. I will be creating a PR that updates marinara for this change shortly after the container builds in the pipeline are completely stable and then update this script.